classes/genre.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :name
+  attr_reader :items, :id
+
+  def initialize(name)
+    @id = Random.rand(1..500)
+    @name = name
+    @items = []
+  end
+
+  def add_item(date, label, author)
+    @items = Item.new(date, label, author)
+    @items.genre = self
```

Comment:
@Mosams  in line 14 you are assigned items to the new item which is not what we want please try to fix the code so you can add item to items array
in line 15 you are selecting genre of items which is an array please be specific and refactor your code so it can add genre automatically